src/otx/tools/converter.py

```diff
@@ -120,7 +120,16 @@
         "task": OTXTaskType.SEMANTIC_SEGMENTATION,
         "model_name": "segnext_b",
     },
-    # ANOMALY_CLASSIFICATION
+    # ANOMALY
+    "ote_anomaly_padim": {
+        "task": OTXTaskType.SEMANTIC_SEGMENTATION,
+        "model_name": "padim",
+    },
+    "ote_anomaly_stfpm": {
+        "task": OTXTaskType.SEMANTIC_SEGMENTATION,
+        "model_name": "stfpm",
+    },
+    # ANOMALY CLASSIFICATION
```

Comment:
I just copied the code from below. I am not sure what the converter is supposed to do.

Reply:
One concern is that the key of this dict should be the same as the model template id in OTX v1 so that it can be found in geti. I guess this needs to be checked.